src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -14,7 +14,9 @@
      * то возвращать 1
      * если не угадал ничего - возвращать 0.</p>
      */
-    public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+    public static int matchResult(int result1, int result2, int bet1, int bet2) {
+        return (result1 == bet1 && result2 == bet2) ? 2 : (result1 > result2 && bet1 > bet2)
+                ? 1 : (result1 < result2 && bet1 < bet2) ? 1
```

Comment:
try to use only one "? 1" construction. hint: use || operator

Reply:
done